<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDialog</class>
 <widget class="QDialog" name="UserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="WatchListWidget"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="Titlelabel">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="TitlelineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Presenterlabel">
       <property name="text">
        <string>Presenter</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="PresenterlineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="SeeWatchListButton">
       <property name="text">
        <string>See Watch List</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DeleteTutorialButton">
       <property name="text">
        <string>Delete tutorial from watch list</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SamePresenterButton">
       <property name="text">
        <string>Tutorials same presenter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>WatchListWidget</zorder>
  <zorder>Titlelabel</zorder>
  <zorder>Presenterlabel</zorder>
  <zorder>TitlelineEdit</zorder>
  <zorder>PresenterlineEdit</zorder>
  <zorder>SamePresenterButton</zorder>
  <zorder>SeeWatchListButton</zorder>
  <zorder>DeleteTutorialButton</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SeeWatchListButton</sender>
   <signal>clicked()</signal>
   <receiver>UserDialog</receiver>
   <slot>SeeWatchList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DeleteTutorialButton</sender>
   <signal>clicked()</signal>
   <receiver>UserDialog</receiver>
   <slot>DeleteTutorial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SamePresenterButton</sender>
   <signal>clicked()</signal>
   <receiver>UserDialog</receiver>
   <slot>SamePresenter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>496</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SeeWatchList()</slot>
  <slot>DeleteTutorial()</slot>
  <slot>SamePresenter()</slot>
 </slots>
</ui>
